modules/monitoring-services.nix

```diff
@@ -149,6 +149,28 @@ in {
         '';
       };
 
+      applicationDataSources = mkOption {
+        type = types.listOf types.attrs;
```

Comment:
The type of this is actually exactly the same as `services.grafana.provision.datasources`. Does anyone know a way I can use the type of an existing option as the type for a new option?

Reply:
You could reference it via `options.services.grafana.provision.datasources.type`

Reply:
Sweet! Thank you.